cfme/tests/infrastructure/test_host.py

```diff
@@ -376,21 +376,22 @@ def test_infrastructure_hosts_refresh_multi(appliance, setup_provider_min_hosts,
                banner where "X" is the number of selected hosts. Properties for each host are
                refreshed. Making changes to test pre-commithooks
     """
+    slice = 2
     hosts_view = navigate_to(provider.collections.hosts, "All")
-    num_hosts = len(hosts_view.entities.get_all())
+    num_hosts = len(hosts_view.entities._current_page_elements)
```

Comment:
Paginator has an attribute for this, I think it would be better than get_all() or private property access.

Reply:
updated to use entities.paginator.items_amount as suggested.